nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -482,6 +482,22 @@ def input_filter_func(point):
         self._fp_inputs[input_id] = input_fp
         return self._fp_inputs[input_id]
 
+    @staticmethod
+    def _get_dynamic_shape(x: List[Tensor]):
```

Comment:
Please provide a context in the docstring, why this function is needed.

Reply:
Done